.github/workflows/merge.yml

```diff
@@ -0,0 +1,69 @@
+name: Trigger release on merge
+run-name: Starting release for ${{ github.actor }} PR merge
```

Comment:
what else could we use? but yeah should also be fine.

Reply:
I mean we can add the commit hash, but that might make the message too long 😄

Reply:
Yeah makes sense. Lets tick with that.